internal/util/testing/grpc_options.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testing
```

Comment:
Is this file copied from other internal areas?  Why can't we just use the existing GRPC stuff?

Reply:
Our existing GRPC stuffs are bind to the config value and `rpc.ServerParams`, I could move this file to `internal/rpc/util.go` and refactor and the rpc library in the future to use this one instead as a TODO.

Reply:
We need to just get this as a generally usable external library sooner rather than later...

Reply:
how about the grpc implementation in other languages? If we choose to externalize this package then we also need to support and maintain these settings in other languages.

Reply:
Just do whatever for this PR, then we'll worry about it later.